src/java.base/share/classes/java/time/LocalTime.java

```diff
@@ -1631,7 +1631,28 @@ public int hashCode() {
      */
     @Override
     public String toString() {
-        StringBuilder buf = new StringBuilder(18);
+        var buf = new StringBuilder(18);
+        formatTo(buf);
+        return buf.toString();
+    }
+
+    /**
+     * Outputs this time into {@code StringBuilder}, such as {@code 10:15}.
```

Comment:
Same here:
```java
/**
 * Prints the toString result to the given buf, avoiding extra string allocations.
 * Requires extra capacity of 18 to avoid StringBuilder reallocation.
 */
```